data_collection/api/tracking_api.yaml

```diff
@@ -343,11 +343,19 @@ paths:
           in: query
           description: |
             Goal ID. Signifies that a goal has been reached and tracks its conversion.
+            Accepts either Goal UUID or legacy integer Goal ID.
 
             Value ``0`` is reserved for E-commerce cart tracking.
           schema:
-            type: integer
-          example: 15
+            type: string
+            format: uuid
+          examples:
```

Comment:
Example doesn't work in this format. Check it [here](https://developers.piwik.pro/en/pptt-3801_goal_uuid/data_collection/api/http_api.html).

See screenshot:
![2022-10-25_11-05](https://user-images.githubusercontent.com/4488774/197732383-89a1da6f-78a5-477f-8e83-461ef699fa5f.png)

Reply:
Yeah - I've noticed. But the syntax I've used is correct (with this OpenApi version at least). Swagged editor compiles it correctly. Can you "do something"  in the repo itself? :)

Reply:
I didn't saw any of this in html when I quickly glanced soooo prolly not. Besides we won't update react library if that's the case for now so I suggest other ways.

Reply:
Gonna additionally check it up tomorrow morning.

Reply:
OK, I'll do some workaround (probably I'll put the legacy `int` version into the description).
I just want to mention that OpenApi 3.1 brought even more useful stuff (like multiple allowed types), it would be nice to utilize it someday 🤞🏻

Reply:
But to be clear: the stuff I used is compliant with 3.0.x

Reply:
Yup, redoc update would be needed to show your examples there. Welp. Sooner than later.

Reply:
Thanks for checking.